lib/pluggable.js

```diff
@@ -0,0 +1,115 @@
+// TODO: make DRY
+
+import * as most from "most";
+
+
+export const CONTINUE = Symbol.for("interlock.continue");
+
+
+function getContext (baseCxt, dependencies) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/940667/8845027/0f35f8d0-30d0-11e5-8f26-3e4aaa9e1dca.png)


This getter has side effects